pkg/controllers/user/alert/initdata.go

```diff
@@ -498,3 +511,65 @@ func getCommonRuleField(groupID, displayName, severity string) v3.CommonRuleFiel
 		TimingField: defaultTimingField,
 	}
 }
+
+type windowsNodeSyner struct {
+	clusterName       string
+	clusterAlertRules v3.ClusterAlertRuleInterface
+	nodeLister        v1.NodeLister
+}
+
+func (l *windowsNodeSyner) Sync(key string, obj *corev1.Node) (runtime.Object, error) {
+	windowsNodes, err := l.nodeLister.List(metav1.NamespaceAll, windowNodeLabel)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to list nodes")
+	}
+
+	if len(windowsNodes) != 0 {
+		return obj, l.initClusterWindowsAlert()
```

Comment:
@aiwantaozi just for my own knowledge, where in rancher code do we create pre-canned/default linux alerts?

Reply:
Hi @alena1108 when registering the all alert related controllers, the default linux alerts created, when windows node added, we created this windows alert